backend/src/apiserver/main.go

```diff
@@ -45,6 +45,7 @@ var (
 	httpPortFlag     = flag.String("httpPortFlag", ":8888", "Http Proxy Port")
 	configPath       = flag.String("config", "", "Path to JSON file containing config")
 	sampleConfigPath = flag.String("sampleconfig", "", "Path to samples")
+	dropMLMDDB       = flag.Bool("drop_mlmd_db", false, "Whether to drop MLMD DB at start up.")
```

Comment:
where do you think is the best place to set this flag to true

Reply:
You mean in the manifests? Like either in lightweight or KF deployment? I'd almost prefer not to check that in since we want to remove it, but I'm open to suggestions.

Reply:
Added it to manifest and env var. PTAL.